packages/widgets/src/builder-widgets-sync.ts

```diff
@@ -0,0 +1,15 @@
+import { Builder } from '@builder.io/react';
+
+import { CarouselComponent } from './components/Carousel';
+import { TabsComponent } from './components/Tabs';
+import { AccordionComponent } from './components/Accordion';
+import { MasonryComponent } from './components/Masonry';
+import { carouselConfig } from './components/Carousel.config';
+import { tabsConfig } from './components/Tabs.config';
+import { accordionConfig } from './components/Accordion.config';
+import { masonryConfig } from './components/Masonry.config';
+
+Builder.registerComponent(CarouselComponent, carouselConfig);
+Builder.registerComponent(TabsComponent, tabsConfig);
+Builder.registerComponent(AccordionComponent, accordionConfig);
+Builder.registerComponent(MasonryComponent, masonryConfig);
```

Comment:
Doesn't seem like this new file is imported/used anywhere, is it?

Reply:
it's meant to be directly imported from dist if desired, in case anyone still needs the old behavior we can point them to importing that directly (it's copy + paste from the old main path, and may still be needed by some if their bundler doesn't support dynamic imports)